recipes/rtd-poetry/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "rtd-poetry" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```

Comment:
Try https://github.com/conda-incubator/grayskull next you send a recipe. It will remove all these unnecessary quote. BTW, I'm pretty sure the quoted URL breaks the bot.